caluma/extensions/permissions.py

```diff
@@ -1,6 +1,28 @@
-from caluma.caluma_core.permissions import BasePermission
+from caluma.caluma_core.mutation import Mutation
+from caluma.caluma_core.permissions import (
+    BasePermission,
+    object_permission_for,
+    permission_for,
+)
 
 
 class MySAGWPermission(BasePermission):
-    def has_permission(self, mutation, info):
-        return True
+    def _admin_or_sagw(self, info):
```

Comment:
Naming: I'd prefer boolean checks to be named `is_*` (or `_is_*` for private methods) to be a bit more explicit

Reply:
Done.